### PR TITLE
Set coredump file's location as /var/log/glusterfs/core_%e.%p

### PR DESCRIPTION
**Pull request by SaravanaStorageNetwork**
Set coredump file's location as /var/log/glusterfs/core_%e.%p

Signed-off-by: Saravanakumar Arumugam <sarumuga@redhat.com>

### CODE REVIEW

**Comment by SaravanaStorageNetwork** [comment]
@humblec  @jarrpa @rtalur PTAL

File: CentOS/gluster-setup.sh [hunk 1]
```diff
@@ -90,6 +90,9 @@ main () {
         echo "heketi-fstab not found"
   fi
 
+  # Set core file location - format: core_<executable_name>.<pid>
+  echo '/var/log/glusterfs/core_%e.%p' | sudo tee /proc/sys/kernel/core_pattern
```
Comment (nixpanic) [inline CentOS/gluster-setup.sh]: Is `sudo` really required? Many of the commands above would not work without having root privileges either.

Comment (SaravanaStorageNetwork) [reply CentOS/gluster-setup.sh]: agreed  - removed sudo

**Comment by SaravanaStorageNetwork** [comment]
@nixpanic  @humblec  There is a concern here - the change done for this specific container will affect the Host setting too.. Is this ok?

**Comment by nixpanic** [comment]
On Wed, Jun 27, 2018 at 11:06:21PM -0700, Saravanakumar Arumugam wrote:
> @nixpanic  @humblec  There is a concern here - the change done for
> this specific container will affect the Host setting too.. Is this ok?   

No, that is not acceptible.

File: CentOS/gluster-setup.sh [hunk 2]
```diff
@@ -90,6 +90,9 @@ main () {
         echo "heketi-fstab not found"
   fi
 
+  # Set core file location - format: core_<executable_name>.<pid>
+  echo '/var/log/glusterfs/core_%e.%p' | tee /proc/sys/kernel/core_pattern
```
Comment (nixpanic) [inline CentOS/gluster-setup.sh]: You would just run `echo /var/.... > /proc/sys/...` and not use `tee`.

**Comment by nixpanic** [comment]
See #93 for a more acceptable approach.

**Pull request closed without merging**